task_reactredux/task1/src/Validation.js

```diff
@@ -0,0 +1,8 @@
+export const validateMail = (email) => {
+    let regExpr = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return regExpr.test(String(email).toLowerCase());
+}
+
+export const validatePassword = (password) => {
+    return password.length > 5;
```

Comment:
move 5 to const, currently, it's a magic number. Same for regexp, you could make it const